communications.md

```diff
@@ -1,89 +1,49 @@
 # Communications
 
-## General guidelines
+## Introduction
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. If done well, we can facilitate great knowledge exchange while still being able to do good work.
 
-## First line of Communication: Texting
+## Tools used for Communication
 
-We use a few tools to help team members communicate with one another: Slack, G Suite and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite and Asana.
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
+Slack is our primary tool for communication where one can message, ping or ask questions to team members in channels depending on projects, interests and teams e.g. #snoke #random #general
 
-#### - Internal & External Messaging Tool: Slack -
-Opt-in: Join channels that you are interested in or obligated to.
+G Suite consists of 1) Gmail, used for client and team correspondence and 2) Calendar, used for setting up meetings by creating an event invitation & booking one of the available rooms.
 
-Setting up Notifications: Once you're part of a channel, set up your notifications to suit your needs:
+Asana is commonly used for project management and any other processes we deem it fit to be used for (e.g Locus Builder Redesign, Office Availability, Weekly Meetings, etc.)
```

Comment:
Shall we also mention the knowledge base part? If it's not an urgent question, they can ask there and we can do weekly clean up.

Reply:
yes good point. will add that in as another process we use for Asana. Perhaps we can even have a specific section about where information can be found (if one has a question) and mention that one can either search in Slack or go to Asana project